pandas/core/dtypes/base.py

```diff
@@ -94,6 +94,18 @@ def is_dtype(cls, dtype):
         except TypeError:
             return False
 
+    @property
+    def _is_numeric(self):
+        # type: () -> bool
+        """
+        Whether columns with this dtype should be considered numeric.
+
+        By default ExtensionDtypes are assumed to be non-numeric.
+        They'll be excluded from operations that exclude non-numeric
+        columns, like groupby reductions, plotting, etc.
```

Comment:
maybe put 'gropuby' in parantheses, as the same holds for normal reductions (in a dataframe)